internal/fnruntime/runner.go

```diff
@@ -108,7 +108,7 @@ type FunctionRunner struct {
 
 func (fr *FunctionRunner) Filter(input []*yaml.RNode) (output []*yaml.RNode, err error) {
 	pr := printer.FromContextOrDie(fr.ctx)
-
+	fr.name = AddDefaultImagePathPrefix(fr.name)
```

Comment:
This will go away (refer to my suggestion).

Reply:
Thanks for suggestion. Updated.